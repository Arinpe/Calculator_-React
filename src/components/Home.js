export default function Home() {
  return (
    <div className="home">
      <h1>Welcome to our page!</h1>
      <p>
        <span>A calculator is a device that performs arithmetic operations.</span>

      </p>

    </div>
  );
}
