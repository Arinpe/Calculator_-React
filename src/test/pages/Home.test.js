import React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import Home from '../../components/Home';

describe('Quote', () => {
  const { asFragment } = render(<Home />);

  test('should render without crashing', () => {
    expect(asFragment()).toMatchSnapshot();
  });
});
