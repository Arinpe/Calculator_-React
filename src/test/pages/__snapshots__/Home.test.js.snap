// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote should render without crashing 1`] = `
<DocumentFragment>
  <div
    class="home"
  >
    <h1>
      Welcome to our page!
    </h1>
    <p>
      <span>
        A calculator is a device that performs arithmetic operations.
      </span>
    </p>
  </div>
</DocumentFragment>
`;
